e2e/README.md

```diff
@@ -30,24 +30,40 @@ helm repo update
 
 You need to install the binary `https://github.com/newrelic/newrelic-integration-e2e-action/tree/main/newrelic-integration-e2e` used in the e2e test
 ```shell
-git clone https://github.com/newrelic/newrelic-integration-e2e-action
-cd newrelic-integration-e2e-action/newrelic-integration-e2e
-go build -o  $GOPATH/bin/newrelic-integration-e2e ./cmd/...
+go install github.com/newrelic/newrelic-integration-e2e-action@latest
+
 ```
 
-You can now run the e2e tests locally once you export the needed variables
+You need New Relic license key (Ingest - License), Api key (User key) and Account before running the tests. More information on how to find these keys, please see [this](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/). 
+
+Set the following environment variables:
 ```shell
-export EXCEPTION_FILE="./1_22-exceptions.yml"
+export EXCEPTION_FILE="1_22-exceptions.yml"
 export LICENSE_KEY=xxx
 export API_KEY=xxx
 export ACCOUNT_ID=xxx
+```
+
+Since some metrics are removed and added depending on the k8s version, the `EXCEPTION_FILE` should point, depending on the k8s version you are testing on, to one of the `/e2e/*-exception.yml` files.
+
+Run the following command to execute the test and make sure that it is ran at the root of the repo:
 
+```shell
 LICENSE_KEY=${LICENSE_KEY} EXCEPTIONS_SOURCE_FILE=${EXCEPTION_FILE}  go run github.com/newrelic/newrelic-integration-e2e-action@latest \
      --commit_sha=test-string --retry_attempts=5 --retry_seconds=60 \
 	 --account_id=${ACCOUNT_ID} --api_key=${API_KEY} --license_key=${LICENSE_KEY} \
 	 --spec_path=./e2e/test-specs.yml --verbose_mode=true --agent_enabled="false"
 ```
 
-Since some metrics are removed and added depending on the k8s version, the `EXCEPTION_FILE` should point, depending on the k8s version you are testing on, to one of the `/e2e/*-exception.yml` files.
+NOTES: 
+For local testing purposes, we usually test against a staging environment. In order to enable testing against staging environment, the following modifications need to be made:
+- Open the the `/e2e/test-specs.yml` and add `--set global.nrStaging=true` to the end of this line `- helm upgrade --install ${SCENARIO_TAG} -n nr-${SCENARIO_TAG} --create-namespace ../charts/newrelic-infrastructure ...` .
```

Comment:
Maybe call out that this modification can apply to multiple lines? There are currently 2 tests defined in `test-spects.yml` that install the integration.

Reply:
👍